### DISCUSSION TITLE
Sheets can be placed anywhere on 4 sides not only in bottom

### DISCUSSION THREAD

**Discussion by dlastnoel** [quasarframework/quasar#17445] [Ideas / Proposals] [answered]
I kinda find bottom sheet to be tightly coupled. For instance bootstrap has this offcanvas feature which is very flexible. I think will be very useful for everybody. Cheers quasar devs! Thank you and stay amazing!

Link fo reference.
https://getbootstrap.com/docs/5.0/components/offcanvas/

**Comment by TobyMosque** [comment, marked as answer]
BottomSheet is just a custom QDialog with a QList, even if that is not documented, all QDialog properties are proxied, so u can configure the `position` and set the `maximized`:

```js
$q.bottomSheet({
  position: 'left',
  maximized: true,
})
```

anyway, i recommend u must look into the QDialog (component and plugin):
https://quasar.dev/vue-components/dialog/
https://quasar.dev/quasar-plugins/dialog

**Reply by dlastnoel** [reply]
Thanks!
